### ISSUE TITLE
Does this support multiple regions and/or accounts?

### ISSUE THREAD

**Issue by NomdeNet** [MGK82/PackTracker#1]
^^

**Comment by MGK82** [comment]
No, it doesn't. As far as I know, HDT doesn't either. ATM I have some other updates in the pipeline. But later on I gonna have a look if this can be realized conveniently for the user. If there is a way to get an ID or anything similar for the current user.

If it's not, we would need something like a profile system. That's doable, but a hazzle for the user. I am not sure if I want something like that.

If the first idea is not available, I gonna have a chat to the HDT devs first what their minds are about multiple regions/acounts.

**Comment by pagem** [comment]
First up, love the idea of this plugin - really useful.

As a player across multiple servers this is essential feature really. I saw the feature request above for support for multiple accounts which you say is in the works so I guess you could use the multple named accounts to support multiple server accounts?

Appreciate this might not be easy but I know that the Collection Tracker plugin supports multiple accounts so wondered if you could team up with them. Thinking a Collection Tracker and Pack Tracker could share some features...

**Comment by MGK82** [comment]
Pack Tracker will distinguish between both, servers and accounts. You will be able to track multiple accounts on the same server and multiple servers for the same account.

**Comment by ScottAllen11** [comment]
Well, it's April 2019 and my pre-release event is tomorrow. Any update on multiple server support and when support for Rise of Shadows will be available? Going to be a shame if my pre-release packs don't get recorded like in December 2018.

**Comment by outgoingzero** [comment]
Hi there

Firstly thank you for this excellent addon.  

I also would like multi region support (as HDT has), as I play on 3 regions.

For now I've added an xml attribute to record the region in History.xml when a pack is opened (so Pack class changed, xml store and fetch methods changed, and AddLogs so I can see region in the log).

Of course this doesn't change pity timers, averages etc as they don't take region in to account and don't rebuild when logging in to different region.

I also tried making the plugin choosing <region>-History.xml but again it wasn't designed for that and seemed to need a lot of changes to make that work.

I'm more a linux/C guy than a C#/WPF guy so I'm not sure of the best way to go about a proper implementation.  Any chance the author who actually know what he is doing might consider implementing it?

Thanks!

**Comment by ScottAllen11** [comment]
If we can get regions working for Descent of Dragons in two weeks, that would be awesome. Currently I have to turn off the addon when I open packs on EU or Asia so my pity timers for NA are right. Quite a hassle, and sometimes I forget. Otherwise one of the best add-ons in HDT.

**Comment by ScottAllen11** [comment]
Any plans to update this? A lot of people play on multiple servers now due to qualifiers. Again, I have to close PityTracker manually every time I update packs on EU/Asia so I don't mess up my pity timers on NA. Then turn it back on manually every time I open packs on NA.
